bittide/src/Bittide/Transceiver.hs

```diff
@@ -16,9 +16,11 @@ import Data.Proxy
 
 transceiverPrbsN ::
   ( KnownNat chansUsed
-  , KnownDomain freeclk
-  , KnownDomain tx
-  , KnownDomain rx
+  , HasSynchronousReset tx
+  , HasDefinedInitialValues tx
+  , HasSynchronousReset rx
+  , HasDefinedInitialValues rx
+  , HasDefinedInitialValues freeclk
```

Comment:
Note that the number of constraints for this function will dwindle once we remove `resetGlitchFilter` from it (future PR).